hamilton/stores/file.py

```diff
@@ -0,0 +1,80 @@
+import shutil
```

Comment:
Calling this `file.py` will overwrite the default `file` by default. Something like `file_store` might be better.

Reply:
AFAIK, there is no `file` in the standard Python library. As you pointed out, this module is internal so the following patterns seem safe 

```python
from hamilton.caching.stores.file import FileResultStore
```  
```python
# hamilton/caching/stores/__init__.py
from .file import FileResultStore

# elsewhere
from hamilton.caching.stores import FileResultStore
```

Let me know if I should prioritize one or another

Reply:
heh, you're right, I'm hallucinating. my bad